src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/ApprenticeshipBase.cs

```diff
@@ -103,8 +103,13 @@ public bool RecognisingPriorLearningExtendedStillNeedsToBeConsidered
                             return false;
                     }
 
-                    if (PriorLearning?.DurationReducedByHours == null || PriorLearning?.PriceReducedBy == null
-                          || PriorLearning?.WeightageReducedBy == null || PriorLearning?.ReasonForRplReduction == null)
+                    if (TrainingTotalHours == null || PriorLearning?.DurationReducedByHours == null 
```

Comment:
This is fine, but just thinking out loud... is RPL for draft apprenticeships only, and if so, should this be in the DraftApprenticeship class and not the base class?